README.mediawiki

```diff
@@ -1,10 +1,6 @@
-People wishing to submit BIPs, first should propose their idea or document to the [https://groups.google.com/g/bitcoindev bitcoindev@googlegroups.com] mailing list (do <em>not</em> assign a number - read <a href="bip-0002.mediawiki">BIP 2</a> for the full process). After discussion, please open a PR. After copy-editing and acceptance, it will be published here.
+People wishing to submit BIPs, should first describe their idea to the [https://groups.google.com/g/bitcoindev bitcoindev@googlegroups.com] mailing list. Please open a pull request to this repository only when substantial progress on the draft has been made, preferably when the draft is nearing completion. After meeting the editorial criteria, a BIP Editor will assign a number, and it will be published here. Authors do <em>not</em> assign a number to their own proposal. Please see <a href="bip-update-process.md">BIP Update Process</a> for the full process.
 
-We are fairly liberal with approving BIPs, and try not to be too involved in decision making on behalf of the community. The exception is in very rare cases of dispute resolution when a decision is contentious and cannot be agreed upon. In those cases, the conservative option will always be preferred.
-
-Having a BIP here does not make it a formally accepted standard until its status becomes Final or Active.
-
-Those proposing changes should consider that ultimately consent may rest with the consensus of the Bitcoin users (see also: [https://en.bitcoin.it/wiki/Economic_majority economic majority]).
+The BIPs repository serves as a publication medium and archive. Having a BIP published here indicates that the proposal is in scope of and has met other formal criteria for this repository, but does not indicate that it is a good idea, has community consensus, or that it is about to be adopted. The BIP Editors are expected to be liberal with approving BIPs, and try not to be too involved in decision making on behalf of the community. Beyond the formal criteria, evaluation of the proposals is left to the audience of the repository. The exception are rare cases when a decision is contentious and cannot be agreed upon. In those cases, the conservative option will be preferred. Authors proposing changes should consider that ultimately consent rests with the Bitcoin users (see also: [https://en.bitcoin.it/wiki/Economic_majority economic majority]).
```

Comment:
The phrase "conservative option" was there before. I had assumed that it meant to decline merging, but actually in the context of being liberal with approval, your read does sound more plausible. I’m leaving this the same for the moment, because I want to mull over whether this should be rephrased and how.

Reply:
I’ve deferred changes to the README to a separate PR #3 as I came to the conclusion that the README should only be updated when this proposal is adopted rather than when it is merged to the repository. I adopted your proposed changes, though.

I have made an attempt to clarify the "conservative option":

```diff
-community consensus, or that it is about to be adopted. The BIP Editors are expected to be liberal with approving BIPs
-and try not to be too involved in decision-making on behalf of the community. Beyond the formal criteria, evaluation of
-the proposals is left to the audience of the repository. The exceptions are rare cases when a decision is contentious
-and cannot be agreed upon. In those cases, the conservative option will be preferred. Authors proposing changes should
-consider that ultimately, consent rests with the Bitcoin users (see also: [https://en.bitcoin.it/wiki/Economic_majority
-economic majority]).
+community consensus, or that it is about to be adopted. The BIP Editors are expected to be liberal with accepting BIPs
+and not to be too involved in decision-making on behalf of the community. Beyond the formal criteria, evaluation of
+proposals is left to the audience of the repository. When a proposal is controversial, but deemed high-quality and in
+scope, the BIP Editors shall err on the side of merging it. Ultimately, consent and adoption rests with the Bitcoin
+users (see also: [https://en.bitcoin.it/wiki/Economic_majority economic majority]).
```